torch/_inductor/scheduler.py

```diff
@@ -360,6 +360,80 @@ def codegen_originating_info(self, buffer, only_once=True):
         buffer.writelines(out_lines)
         self.written = True
 
+    def get_estimated_runtime(self, rwbytes: int) -> float:
+        layout = None
+        if not self.node:
+            assert(self.snodes)
+            layout = self.snodes[0].node.get_layout()
+        else:
+            layout = self.node.get_layout()
+
+        if "cuda" != layout.device.type:
+            # default to no reordering based on runtime
+            return 0
+
+        from .ir import CollectiveKernel, ComputedBuffer
+
+        # TODO(xmfan): figure out how to get hardware specs, use A100 for now
+        gpu_memory_bandwidth = 1555 * 2**30  # 1555 GBps
+        gpu_flops = 312**12  # 312 TFLOPS
```

Comment:
Reuse existing get_flops function (hardware agnostic), and factor it out into utils.py ? 

https://github.com/pytorch/pytorch/blob/main/torch/_inductor/fx_passes/pad_mm.py#L151-L165

That also shows memory bandwidth call `get_dram_gbps`

Reply:
Addressed in 
a9bb8ea